app/uk/gov/hmrc/hmrcemailrenderer/templates/offpayroll/offpayrollPrivateBetaInvitation.scala.html

```diff
@@ -1,22 +1,57 @@
+
+    <p style="margin: 0 0 30px; font-size: 19px;">It could be used by:</p>
 
-    <p style="margin: 0 0 30px; font-size: 19px;">Don’t delete this email, as you’ll need to use this link each time you want to access the service.
+    <p style="margin: 0 0 30px; font-size: 19px;">
+        <ul>
+            <li>a worker providing a service</li>
+            <li>a person or company hiring a worker</li>
+            <li>an agency that is paying a worker</li>
+        </ul>
     </p>
 
-    <p style="margin: 0 0 30px; font-size: 19px;"><h2>Why you got this email</h2></p>
+    <p style="margin: 0 0 30px; font-size: 19px;">This version is just for testing purposes so <b>you can’t rely on the results it gives.</b> You should carry on using existing methods for checking contracts.</p>
+
+    <p style="margin: 0 0 30px; font-size: 19px;">We'll be adding more features to the next version of the service.</p>
+
+    <p style="margin: 0 0 30px; font-size: 19px;"><h2>How to start testing</h2></p>
+
+    <p style="margin: 0 0 30px; font-size: 19px;">This link will take you straight to the test version of the service:</p>
+
+    <p style="margin: 0 0 30px; font-size: 19px;"><a href="@{params("verificationLink")}" style="color: #005EA5;">@{params("verificationLink")}</a></>
+
+    <p style="margin: 0 0 30px; font-size: 19px;"><b>Please don't share this link with other people.</b></p>
+
+    <p style="margin: 0 0 30px; font-size: 19px;">We'll be monitoring how the service is used by invitees and may need to restrict access.</p>
+
+    <p style="margin: 0 0 30px; font-size: 19px;"><h2>How to give us your feedback</h2></p>
+
+    <p style="margin: 0 0 30px; font-size: 19px;">If you want to comment on any question please click the feedback link above it. This will open a form for you to tell us what you think. </p>
+
+    <p style="margin: 0 0 30px; font-size: 19px;">The more feedback you give us, the better we can make the nest version of this service.</p>
+
+    <p style="margin: 0 0 30px; font-size: 19px;"><h2>How long testing lasts</h2></p>
+
+    <p style="margin: 0 0 30px; font-size: 19px;">You can try this service up to 60 times during the next 30 days.
+    
+    <p style="margin: 0 0 30px; font-size: 19px;">After that we'll make and test a second version of the service based on the feedback we get.</p>
 
-    <p style="margin: 0 0 30px; font-size: 19px;">You gave this contact email address when you signed up for the Off Payroll private beta.</p>
+    <p style="margin: 0 0 30px; font-size: 19px;">Thank you for your help and feedback,</p>
 
-    <p style="margin: 0 0 30px; font-size: 19px;">From HMRC Off Payroll</p>
+    <p style="margin: 0 0 0px; font-size: 19px;"><b>Rhys Hobbs</b></p>
+    <p style="margin: 0 0 0px; font-size: 19px;">Digital Service Manager</p>
+    <p style="margin: 0 0 0px; font-size: 19px;">Personal Tax Customer Product and Process</p>
 
```

Comment:
Should the sender of the email be a department rather than an Individual? If this is a customer facing email, is Digital Service Manager a customer facing role?